modules/thermal_hydraulics/test/tests/base/simulation/tests

```diff
@@ -17,12 +20,14 @@
 \s+hx:secondary
 \s+inlet
 \s+outlet"
+    requirement = 'The system shall be able to output the component loops to console.'
   []
 
   [err.no_smp]
     type = 'RunException'
     input = 'err.no_smp.i'
     expect_err = "Single matrix preconditioning with full coupling is required to run\. Please, check that your input file has the following preconditioning block\:"
     deleted = "#190"
+    requirement = 'The system shall error if single matrix preconditioning is not in use in a thermal hydraulics simulation.'
```

Comment:
I think we should do "report an error" instead of "error", since "error" sounds like something you don't want your system to do :) Same comment for your other requirements

Reply:
interesting both are used approximately in the same quantity
```
(moose) ~/projects/moose_v1/modules$ ggrep -r "The system shall report an error" | wc -l
      39
(moose)~/projects/moose_v1/modules$ ggrep -r "The system shall error" | wc -l
      34
(moose) ~/projects/moose_v1/modules$ cd ../test/
(moose) ~/projects/moose_v1/test$ ggrep -r "The system shall error" | wc -l
^[[A^[[A^[[A     142
(moose) ~/projects/moose_v1/test$ ggrep -r "The system shall report an error" | wc -l
     140
(moose) ~/projects/moose_v1/test$ 
```

Reply:
I can change though

Reply:
Yeah "report an error" is preferable going forward

Reply:
alright well I changed the repo so we do not go backward